### ISSUE TITLE
Does val-objectness increase mean overfitting?

### ISSUE THREAD

**Issue by yxNONG** [ultralytics/yolov5#114]
i try to training with different setting in my custom small dataset (880 img for train and 220 for test) in which the classes are unbalanced.

question:
1.in my result, the val-objectness of both models is decreased first and then increased 
   does it means overfitting?
2.seem like the result of yolo5-s and yolo5-l is similar, for better ap, should i try yolo5-x?
   or any suggestion like use Ciou or smaller learning rate？ 

outcome:
model=yolov5-s, batch_size=16, lr=0.01, epoch=300:
<img width="357" alt="s-600-lr0 01-fl0 0" src="https://user-images.githubusercontent.com/62932917/84905615-767a1b00-b0e3-11ea-847f-66719d7626d4.png">

model=yolov5-l, batch_size=32, lr=0.01, epoch=300, fl_gamma=1.5:
<img width="356" alt="l-300-lr0 01-fl1 5" src="https://user-images.githubusercontent.com/62932917/84905759-a9241380-b0e3-11ea-8a7d-385d1613f89a.png">

model=yolov5-l, batch-size=32, lr=0.02, epoch=600, fl-gamma=1.5:
<img width="355" alt="l-600-lr0 02-fl1 5" src="https://user-images.githubusercontent.com/62932917/84905830-c527b500-b0e3-11ea-84e1-a7764d0523d0.png">

**Comment by glenn-jocher** [comment]
@yxNONG all I'll say is if you want to delay overfitting, don't load pretrained weights.

**Comment by glenn-jocher** [comment]
@yxNONG also, download the latest version, AutoAnchor may help

**Comment by enoceanwei** [comment]
@yxNONG 

Hi, have you solved the problem? I am suffering the same issues just like you, could you tell me how to avoid the overfitting during the training, many thanks.

Kind regards
